spec/type_aliases/loglevel_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe 'Apache::LogLevel' do
+  [
+    'info',
+    'warn ssl:info',
+    'warn mod_ssl.c:info',
+    'warn mod_ssl.c:info',
+    'warn ssl_module:info',
+    'trace4',
+  ].each do |allowed_value|
+    it { is_expected.to allow_value(allowed_value) }
+  end
+
+  [
```

Comment:
How about `xemerg` to test if `^` at the start is needed?

Reply:
The regex is copied from the the function and is very lax and could be better.  I thought about trying to make it stricter, but decided that a change like that might be suited to a separate PR.

As things stand a `^` at the beginning would be bad. The apache docs describe the allowed syntax as...

> Syntax
> LogLevel [module:]level [module:level] ...

I think in a perfect world, $log_level would have actually accepted an `Array`. I doubt it's worth introducing that sort of change now though.